source/develop/release-management/features/network/ipv6_subnets.html.md

```diff
@@ -0,0 +1,179 @@
+
+* dhcpv6-stateless: IP addresses are assigned by OVN's IPv6 stack, based on the port's mac address and the advertised prefix - in the RA messages.The connection parameters (MTU, max hops, DNS, etc) are configured through the subnet's native DHCPv6 service.
+
+* dhcpv6-stateful: Both the IP addresses and the connection parameters are extraced from the router RA messages.
+
+### IPAM configuration requirements
+
+* slaac: this IPAM configuration does not rely on OVN's subnet DHCP service, which means a DHCP_Options object is **not** required. The RA messages are sent by an OVN Logical_Router, thus one is obviously required.
+
+* dhcpv6-stateless: this IPAM configuration relies on OVN's subnet DHCP services to get its IP address, and on the Logical_Router's RA message's connection parameters to properly configure the connection. Thus, a DHCP_Options entry is required, as is a Logical_Router object.
+
+* dhcpv6-stateful: this IPAM configuration relies solely in OVN's subnet DHCP services, and is very close to the IPv4 configuration counterpart - a DHCP_Options object is required, having both an IPv6 cidr, and ip_version set to IPv6.
+
+### IPAM configuration
+
+This section explains how to configure each of the IPAM options. They are listed by development priority.
+
+* dhcpv6-stateful
+  - create an ovn DHCP_Options object, having an ipv6 cidr configured.
+    It requires the user to specify the ethernet address for the DHCPserver
+    to use in the *options:server_id* DHCP_Options attribute.
+  - to enable IPAM for IPv6, the logical switch (ovn entity) where the ports are
+    created has to include the **other_config: ipv6_prefix** key defined.
+    The value should be the the IP prefix featured in the provisioned cidr
+    parameter, ending in '::' - e.g. 8230:5678::, for instance.
+
+* slaac
+  - configure the logical router port(s) to operate in *slaac* mode.
+    To do that, the logical router port *ipv6_ra_configs:address_mode* key
+    should be set to ***slaac***.
+
+* dhcpv6-stateless
+  - configure the logical router port(s) to operate in *dhcpv6-stateless* mode.
+    The logical router port **ipv6_ra_configs:address_mode** key should be set
+    to ***dhcpv6-stateless***.
+  - configure the DHCP_Options object to operate stateless, by setting the
+    **options : dhcpv6_stateless** key's value to '**yes**'
+
+### Alignment with networking API
+
+The ovirt-provider-ovn provides an opinionated subset of OpenStack's networking API v2. It it's [subnet definition](https://developer.openstack.org/api-ref/network/v2/?expanded=create-subnet-detail#create-subnet), it can be seen that the API features 2 parameters to influence IPv6 related configurations: **ipv6_ra_mode**, and **ipv6_address_mode**. Both of those attributes accept the following values:
```

Comment:
It it's not so clear

Reply:
hm, it literally says: 

Specify the ipv6_ra_mode and ipv6_address_mode attributes to create subnets that support IPv6 configurations, such as stateless address autoconfiguration (SLAAC), DHCPv6 stateful, and DHCPv6 stateless configurations.

Not sure what else I can do to make this more understandable